src/util.cpp

```diff
@@ -955,3 +955,10 @@ fs::path AbsPathForConfigVal(const fs::path& path, bool net_specific)
 {
     return fs::absolute(path, GetDataDir(net_specific));
 }
+
+void ScheduleBatchPriority(pid_t tid) {
```

Comment:
pid_t is not available on windows, I think it would be better to remove the parameter as it's always passed as 0.

Reply:
I think you're right.